LSP-pyright.sublime-settings

```diff
@@ -32,9 +32,7 @@
 		"python.analysis.diagnosticSeverityOverrides": {
 			"reportDuplicateImport": "warning",
 			"reportImplicitStringConcatenation": "warning",
-			"reportMissingParameterType": "none",
 			"reportUnboundVariable": "warning",
-			"reportUninitializedInstanceVariable": "none",
```

Comment:
removed those two since those are default values and I guess the intention is to only include overrides here